provider/resource_collection.go

```diff
@@ -368,6 +480,15 @@ func collectionApiCreate(ctx context.Context, d *schema.ResourceData, m interfac
 	return api.Create(ctx)
 }
 
+func collectionApiWaitAndRead(ctx context.Context, d *schema.ResourceData, m interface{}) (api *models.CollectionApi, err error) {
+	tflog.Info(ctx, "RSRC COLLECTION:   read ("+d.Get("name").(string)+")")
+	// To read a collection, we use...
+	//   storage/collections/config/{collection} -- GET
+	api, _ = collection(ctx, d, "collection_config", m)
```

Comment:
shouldn't we handle this error?

Reply:
ty, copied from collectionApiRead and didn't notice